test/set/jsonGraphs/expired.spec.js

```diff
@@ -130,7 +130,7 @@ describe("an expired value", function() {
             }]
         );
 
-        expect(optimizedPaths.length).to.equal(0);
```

Comment:
Could we rename this from `optimizedPaths` to `successfulPaths`?  It can be a bit confusing.